cfme/utils/appliance/__init__.py

```diff
@@ -2830,6 +2830,10 @@ class DummyAppliance(object):
     build = 'missing :)'
     managed_known_providers = []
 
+    def __attrs_post_init__(self):
+        from cfme.modeling.base import EntityCollections
```

Comment:
**REQUIRED** `collections = attr.ib(init=False, repr=False, default=attr.Factory(EntityCollections.for_appliance, takes_self=True)` should be used imho

it may be necessary to wrap the import in a function due to cycle-avoidance